bazel_tools/scala_version.bzl

```diff
@@ -35,6 +35,9 @@ scala_artifacts = {
     },
 }
 
+scalapb_version = "0.11.3"
+scalapb_protoc_version = "0.9.2"
```

Comment:
Is there a good reason why these should be in here rather than in `bazel_tools/scala.bzl`? It doesn’t seem like they depend on the scala version.

Reply:
It's used in the `WORKSPACE`. Can we move it and still use it there?

Reply:
I don’t see why we shouldn’t be able to do that.

Reply:
Got some cycles, but moving them into their own file (_scalapb.bzl_) works well.